daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/ex_nums.dat.xml.runtime2.error

```diff
@@ -20,7 +20,7 @@
   <array>
     <be_bool16>false</be_bool16>
     <be_bool16>true</be_bool16>
```

Comment:
One concern with changing the extension to `.runtime1`, `.runtime2`, `.error`, etc. extensions is that editors are going to get confused about the file type. I think I prefer the original names, or at least a naming convention where the final extension correctly identifies the file type.